### PR TITLE
Download GeoJSON when available

### LINKED ISSUE #255: Download button should link to GeoJSON format when available
Since v6.6, the EMS manifest has support for layers having multiple geospatial formats such as GeoJSON and TopoJSON. When multiple formats exist for a layer, the Download button should always try to use the more interchangeable GeoJSON format falling back to the default format (usually TopoJSON) when GeoJSON is not available. The map display should always try to use the TopoJSON format falling back to GeoJSON when TopoJSON is not available.

### PR DESCRIPTION
**Pull request by nickpeihl**
Fixes #255 

The "Download <format type>" button tries to get the "geojson" format of the layer. If a "geojson" format is unavailable, the default format type is used.

This requires https://github.com/elastic/ems-client/pull/53.

### CODE REVIEW

File: public/js/components/feature_table.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ export class FeatureTable extends Component {
 
   _renderToolsRight() {
     let humanReadableFormat;
-    const format = this.props.config.getDefaultFormatType();
+    const format = this.props.config.getFormatOfType('geojson');
```
Comment (jsanz) [inline public/js/components/feature_table.js:86]: Where is this function defined? I can't find it in EMS Client [file_layer.ts](https://github.com/elastic/ems-client/blob/master/src/file_layer.ts) :sweat:

Comment (jsanz) [reply public/js/components/feature_table.js:86]: Sorry, just checked in the wrong order :upside_down_face: this is here https://github.com/elastic/ems-client/pull/53

**Comment by jsanz** [comment]
@nickpeihl I've updated this PR to use EMS 7.11, let me know if there's anything else to do here but from what I see this is ready to be merged.

I checked the issue of hitting the download button not triggering a file download but a new tab open. The [`download`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download) directive on the button is not working for URLs from different domains on [Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=714373) and [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=874009).

Maybe for the future, we could think of a refactor of the app to download the datasets into the `localStorage` to avoid repeated downloads, and then the logic of the download button to take the resource from there.

**Comment by nickpeihl** [comment]
> @nickpeihl I've updated this PR to use EMS 7.11, let me know if there's anything else to do here but from what I see this is ready to be merged.

I've marked the PR as ready to review. If you are ok with it, can you give a `lgtm` review before we merge?

> I checked the issue of hitting the download button not triggering a file download but a new tab open. Unfortunately, the [`download`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attr-download) directive on the button is not working for URLs from different domains on [Chrome](https://bugs.chromium.org/p/chromium/issues/detail?id=714373) and [Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=874009) so we cannot use it here.

Yeah, I think this is unavoidable for now. 

> Maybe for the future, we could think of a refactor of the app to download the datasets into the `localStorage` to avoid repeated downloads, and then the logic of the download button to take the resource from there.

I like this idea! 🤔 Although I think [IndexedDB](https://developer.mozilla.org/en-US/docs/Web/API/IndexedDB_API) is better suited for storing larger datasets. Let's revisit this in the future.

**Review by jsanz — APPROVED** [review]
LGTM

**Pull request merged**